src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Shapes/Shapes_Tests.cs

```diff
@@ -20,60 +20,60 @@ public partial class Shapes_Tests : SampleControlUITestBase
 		[AutoRetry]
 		public void Draw_polyline()
 		{
-			Run("SamplesApp.Windows_UI_Xaml_Shapes.PolylinePage");
+			Run("SamplesApp.Windows_UI_Xaml_Shapes.PolylinePage",skipInitialScreenshot:true);
 			_app.WaitForElement("DPolyline");
-			TakeScreenshot($"PolylinePage");
+			TakeScreenshot($"PolylinePage",ignoreInSnapshotCompare:true);
 			TabWaitAndThenScreenshot("ChangeShape");
 
 			void TabWaitAndThenScreenshot(string buttonName)
 			{
 				_app.Marked(buttonName).FastTap();
 				_app.WaitForElement("DPolyline");
-				TakeScreenshot($"PolylinePage - {buttonName}");
+				TakeScreenshot($"PolylinePage - {buttonName}", ignoreInSnapshotCompare: true);
 			}
 		}
 
 		[Test]
 		[AutoRetry]
 		public void Draw_polygon()
```

Comment:
I don't quite see why this one shouldn't be stable for screenshot comparison. @jeromelaban ?

Reply:
It does not feel like it indeed. I may be that it was incorrectly added to the original issue. We should check in the build screenshot tests if it's really appearing there.